torch/csrc/jit/passes/normalize_ops.cpp

```diff
@@ -127,6 +127,7 @@ const std::unordered_map<Symbol, Symbol>& getOperatorAliasMap() {
       {aten::special_xlogy, aten::xlogy},
       {aten::special_log_softmax, aten::log_softmax},
       {aten::orgqr, aten::linalg_householder_product},
+      {aten::adjoint, aten::mH},
```

Comment:
why wasn't `mT` included here?

Reply:
because `mT` is not an alias of any other function.

Reply:
"adjoint" might be the more readable term than "mH" but whatever seems fine

Reply:
One problem of doing it that way is that I cannot add `mH` as an alias in the `OpInfo`. Leaving it like this for now.